framework/spree/wishlist/use-wishlist.tsx

```diff
@@ -0,0 +1,93 @@
+import { useMemo } from 'react'
+import type { SWRHook } from '@commerce/utils/types'
+import useWishlist from '@commerce/wishlist/use-wishlist'
+import type { UseWishlist } from '@commerce/wishlist/use-wishlist'
+import type { GetWishlistHook } from '@commerce/types/wishlist'
+import type { IToken } from '@spree/storefront-api-v2-sdk/types/interfaces/Token'
+import type { GraphQLFetcherResult } from '@commerce/api'
+import type { Wishlist } from '@spree/storefront-api-v2-sdk/types/interfaces/Wishlist'
+import ensureIToken from '../utils/tokens/ensure-itoken'
+import normalizeWishlist from '../utils/normalizations/normalize-wishlist'
+import isLoggedIn from '@framework/utils/tokens/is-logged-in'
```

Comment:
We should always do relative paths inside the providers folder, otherwise build will fail for other providers